tools/cli/internal/openapi/filter/operations.go

```diff
@@ -35,7 +35,6 @@ func (f *OperationsFilter) Apply() error {
 		for _, operation := range pathItem.Operations() {
 			if operation.Extensions != nil {
 				delete(operation.Extensions, "x-xgen-owner-team")
-				delete(operation.Extensions, "x-xgen-changelog")
```

Comment:
is this line the fix? Do we plan to include this extension for all the specs?

Reply:
yes, and I plan to keep it in all specs, I added the removal then caused a regression